packages/cozy-client/src/models/paper.js

```diff
@@ -1,5 +1,6 @@
 import add from 'date-fns/add'
 import sub from 'date-fns/sub'
+import get from 'lodash/get'
```

Comment:
We avoid to use `lodash/get` now and use [Nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)